lib/env.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+module.exports = (
+  typeof process === "object" && process !== null &&
+  typeof process.env === "object" && process.env !== null &&
+  process.env
+) || {};
```

Comment:
Is the `process` guard still needed here? I don' think it's because of PhantomJS.
Maybe some Meteor build thing?